openshift_cli_installer/manifests/clusters.example.yaml

```diff
@@ -97,7 +97,7 @@ clusters:
 - name: aws-ipi-c1
   base-domain: aws.domain.example.com
   platform: aws
-  region: us-east-2
+  region: us-east-2 # optional, if not provided, region will be selected by automation which have the least number of VPCs.
```

Comment:
user must pass `--aws-auto-region` to get auto region select, also needed to add to the example yaml.


I think auto-region should be per cluster and not global property.

Let's say I have 10 clusters to deploy, once you select the region as top level property we probably fail since all 10 clusters will be in the region.

Reply:
I see region is set per cluster as it should, so we only need to accept auto-region per cluster and not as global parameter.

Reply:
Okay,  we can add `auto-region` param per cluster